app/src/main/java/org/openobservatory/ooniprobe/adapter/TestsListAdapter.java

```diff
@@ -42,58 +47,75 @@ public TestsListAdapter(MainActivity context, ArrayList<NetworkMeasurement> valu
     public TestsListAdapter.ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
         View v;
         switch (viewType) {
-            case 0:
-                v = LayoutInflater.from(parent.getContext()).inflate(R.layout.row_pending_test, parent, false);
-                break;
-            case 1:
-                v = LayoutInflater.from(parent.getContext()).inflate(R.layout.row_finished_test, parent, false);
-                break;
             default:
-                v = LayoutInflater.from(parent.getContext()).inflate(R.layout.row_pending_test, parent, false);
+                v = LayoutInflater.from(parent.getContext()).inflate(R.layout.row_finished_test, parent, false);
```

Comment:
I don't think the test row will always be this, I believe there will me more prototype like there were before so I left the switch only in this adapter and not in the other two